app/layout/project/navigation/breadcrumbs/index.tsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+
+import { useRouter } from 'next/router';
+
+import { useProject } from 'hooks/projects';
+import { useScenario } from 'hooks/scenarios';
+
+import { cn } from 'utils/cn';
+
+export const Breadcrumbs = ({ className }: { className?: string }): JSX.Element => {
+  const { pathname, push, query } = useRouter();
+  const { pid, sid } = query as { pid: string; sid: string };
+  const projectQuery = useProject(pid);
+  const scenarioQuery = useScenario(sid);
+
+  return (
+    <nav className="flex font-sans text-xs text-white" aria-label="Breadcrumb">
+      <ol role="list" className="flex items-center space-x-2">
+        <li>
+          <button
+            onClick={() => push('/projects')}
+            className={cn({
+              'flex h-6 items-center overflow-hidden hover:text-primary-500 focus:outline-none':
+                true,
+              'font-semibold text-primary-500': pathname === '/projects',
+              [className]: !!className,
```

Comment:
It's weird the `className` the component receives affects exclusively this very element, I would expect to apply it on line 17 so I can adapt the breadcrumbs to other contexts if needed but not here.